lib/ice_cube/parsers/ical_parser.rb

```diff
@@ -83,5 +92,20 @@ def self.rule_from_ical(ical)
 
       Rule.from_hash(params)
     end
+
+    private
+
+    def self.find_zone(tzid)
+      (_, zone) = tzid&.split("=")
+      begin
+        Time.find_zone!(zone) if zone.present?
+      rescue ArgumentError
+        (rails_zone, _tzinfo_id) = ActiveSupport::TimeZone::MAPPING.find do |(k, _)|
+          Time.find_zone!(k).now.strftime("%Z") == zone
```

Comment:
This is where the time zone identifiers like `"MDT"` are matched with Rails time zones, using `strftime`.